torch/_inductor/ir.py

```diff
@@ -2185,7 +2195,11 @@ def create(  # type: ignore[override]
         )
         scan_type = Scan
         if num_splits > 1:
-            supports_split = torch.version.hip is None and len(dtypes) == 1
+            supports_split = (
+                has_triton
+                and torch.version.hip is not None
+                and triton_version >= "3.3.0"
+            ) or (len(dtypes) == 1)
```

Comment:
This looks like it changes behavior - should it be something like the following?

```suggestion
            supports_split = (
                torch.version.hip is None
                or (has_triton and triton_version >= "3.3.0")
            ) and (len(dtypes) == 1)
```

(e.g., for triton 3.2, `len(dtypes) == 1` and `torch.version.hip is not None` will now result in `supports_split = True`, whereas before previously it would be `False`)

Reply:
@jataylo https://github.com/triton-lang/triton/pull/6127 was merged into main and https://github.com/triton-lang/triton/pull/6129 was merged into the 3.3 release branch